### ISSUE TITLE
FixedResourcePool failed

### ISSUE THREAD

**Issue by secmask** [xetorthio/jedis#5]
I'm using jedis version on your git repository, and i have some problem with its ConnectionPool, I think RepairThread should catch exception that throw by createResource. I make a test like this:
1. new JedisPool() <--> RedisServer did not started
2. getResource()   <--> RedisServer still down.
3. getResource()   <--> RedisServer up.
4. getResource()   <--> RedisServer down.
5. getResource()   <--> RedisServer up.

in summary, a loop try to get connection to RedisServer, put an item, server side will up/down redis-server for several times, and I hope JedisPool will wait for connection available. in my test, JedisPool turn frozen after some restart of redis-server.
